lib/matplotlib/table.py

```diff
@@ -718,8 +718,8 @@ def table(ax,
         A bounding box to draw the table into. If this is not *None*, this
         overrides *loc*.
 
-    edges : substring of 'BRTL' or {'open', 'closed', 'horizontal', 'vertical'}
-        The cell edges to be drawn with a line. See also
+    edges : substring of 'BRTL' or {'open', 'closed', 'horizontal', 'vertical'},
+        default: 'closed'. The cell edges to be drawn with a line. See also
```

Comment:
```suggestion
    edges : substring of 'BRTL' or {'open', 'closed', 'horizontal', 'vertical'}, \
default: 'closed'. 
        The cell edges to be drawn with a line. See also
```
I think that this will render the default on the same line as the options.